src/utils/date.ts

```diff
@@ -1,7 +1,25 @@
+const now = new Date()
+
 export const safeDateObj = (date: Date | string | number | undefined) => {
   if (!date) return undefined
   if (typeof date === 'string' && /^\d+$/.test(date)) return new Date(parseInt(date))
   const dateObj = date instanceof Date ? date : new Date(date)
   if (Number.isNaN(dateObj)) return undefined
   return dateObj
 }
+
+export const secondsToDate = (seconds: number) => new Date(seconds * 1000)
+
+export function secondsToDateInput(seconds: number) {
+  const date = secondsToDate(seconds) // Convert seconds to milliseconds
+  const year = date.getFullYear()
+  const month = String(date.getMonth() + 1).padStart(2, '0') // Month is zero-indexed
+  const day = String(date.getDate()).padStart(2, '0')
+  return `${year}-${month}-${day}`
+}
+
+export const getSecondsFromDate = (date: Date) => {
+  // @ts-ignore typescript doesn't support date operators
+  const value = Math.floor((date - now) / 1000)
+  return value
+}
```

Comment:
given that this function is only used once, probably it's better to just use a literal expr at where it's used?